tests/test_local_eval.py

```diff
@@ -0,0 +1,243 @@
+from dataclasses import fields
+from typing import Dict, Tuple, cast
+from unittest import mock
+
+import pytest
+
+from flagsmith import Flagsmith
+from flagsmith.models import Flag, Flags
+
+
+@pytest.fixture
+def api_key(request: pytest.FixtureRequest) -> str:
+    return cast(str, request.config.option.api_key)
+
+
+@pytest.fixture
+def api_url(request: pytest.FixtureRequest) -> str:
+    return cast(str, request.config.option.api_url)
+
+
+@pytest.fixture
+def is_migrated(request: pytest.FixtureRequest) -> bool:
+    return cast(bool, request.config.option.migrated)
+
+
+@pytest.fixture
+def identifiers() -> Tuple[str, ...]:
+    return ("1", "2", "3", "4", "5")
+
+
+def get_matching_str(*values: str) -> str:
+    class _matching_str(str):
+        _values = values
+
+        def __eq__(self, other: object) -> bool:
+            return other in self._values
+
+    return _matching_str(" / ".join(values))
+
+
+def _get_flag_mock(**kwargs: object) -> mock.Mock:
+    mock_kwargs = {field.name: mock.ANY for field in fields(Flag)}
+    mock_kwargs.update(kwargs)
+    return mock.MagicMock(
+        spec=Flag(mock.ANY, mock.ANY, mock.ANY, mock.ANY), **mock_kwargs
+    )
+
+
+@pytest.fixture
+def environment_default_or_percentage_split_value_flag_mock() -> mock.Mock:
+    return _get_flag_mock(
+        value=get_matching_str(
+            "environment default",
+            'overridden by "split_segment"',
```

Comment:
If we can, I'd like to be more specific here. Whether the identity belongs to the percentage split segment shouldn't change before and after migration. I _think_ this is going to mean that we're just checking that it takes one of the 2 possible values.

Reply:
Switched to static values.